cookie-banner-rules-list.json

```diff
@@ -5240,6 +5239,28 @@
       "cookies": {},
       "id": "8308357f-6a66-433d-bfc3-de401410c350",
       "domains": ["hubspot.com"]
+    },
+    {
+      "id": "05157ed1-12c2-4f84-9dff-718fae5bc096",
+      "domains": ["reverso.net"],
+      "cookies": {
+        "optIn": [
+          {
+            "name": "euconsent-v2",
+            "value": "CRolnUARolnUAAHABBENDGCgAP_AAH_AAAAAJdtf_X__b2_r-_7_f_t0eY1P9_7__-0zjhfdl-8N3f_X_L8X52M7vF36tq4KuR4ku3LBIUdlHPHcTVmw6okVryPsbk2cr7NKJ7PEmnMbO2dYGH9_n1_z-ZKY7___f__z_v-v________7-3f3__5___-__e_V__9zfn9_____9vP___9v-_9__________3_79_7_H9-f_87gl2ASYatxAF2JY4E20YRQogRhWEhVAoAKKAYWiAwgdXBTsrgJ9YRIAUAoAjAiBDgCjJgEAAAEASEQASBHggEAAEAgABAAqEQgAI2AQUAFgIBAAKA6FijFAEIEhBkRERCmBARIkFBPZUIJQf6GmEIdZYAUGj_ioQESgBisCISFg5DgiQEvFkgWYo3yAEYAUAolQrUEnpoAFjI2AAA.f_gAD_gAAAAA"
```

Comment:
I'm worried that this is a highly dynamic cookie that may even include user identifiers. we should only set it if we can rule this out.

Reply:
You are right. I should have thought about this earlier :/ Would it be okay to have only an opt-out cookie rule and only an opt-in click rule?

Reply:
That would be okay. I'm wondering though again how dynamic that cookie is, and whether it would work for the other didomi sites too.

Reply:
I'm afraid it won't be possible to satisfy everyone with one single trick, but it may still be worth a try. If you haven't figured it out yet, you can take a look at the following repository: https://github.com/InteractiveAdvertisingBureau/GDPR-Transparency-and-Consent-Framework which contains the technical specification for the Transparency and Consent Framework (TCF) that Didomi cookies comply with.

For now, I have removed the consent cookie.